src/assets/styles/components/SideBar.scss

```diff
@@ -9,6 +9,6 @@
   position: fixed;
   width: 31.21%;
   height: 100vh;
-  background: rgba(0, 0, 0, 0.09);
+  background: #e6e6e6;
```

Comment:
nitpick: would this also be a design token?

Reply:
`#e6e6e6` is missing. Needed an update for design token. I will contacting Scott to update the right color.

Reply:
--color-g-200

Reply:
The functionality looks good. I dropped in the background color for your from Blocks

Reply:
@robertwollny-pnmac  I tried that g-200 it seems color doesn’t match with the design specks

Reply:
If it doesn't match the design specs, I'd add a // TODO note in the code base, and make sure @sfowles  is aware.